### PR TITLE
[PREPARES FOR JENKINS-36871] Matching changes to PR jenkins#2480

### PR DESCRIPTION
**Pull request by stephenc**
See [Jenkins PR #2480](https://github.com/jenkinsci/jenkins/pull/2480) this is a supporting change for [JENKINS-36871](https://issues.jenkins-ci.org/browse/JENKINS-36871)

@reviewbybees @jenkinsci/code-reviewers

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,19 +12,53 @@
...
   </scm>
 
-  <licenses>
-    <license>
-      <name>MIT License</name>
-      <url>http://www.opensource.org/licenses/mit-license.php</url>
-    </license>
-  </licenses>
+  <properties>
+    <!-- TODO remove once Jenkins core 2.16 released -->
```
Comment (amuniz) [inline pom.xml]: Why? if this is removed then the parent matching version will be used (which is `1.625.3` currently), so how is it going to pick anything from 2.16?

Comment (stephenc) [inline pom.xml]: Should read update to 2.16

Comment (amuniz) [inline pom.xml]: It might be my broken english, but I would never have read that... 😄

**Comment by amuniz** [comment]
:bee:

File: pom.xml [hunk 2]
```diff
@@ -10,19 +12,53 @@
   <artifactId>instance-identity</artifactId>
   <version>2.1-SNAPSHOT</version>
   <packaging>jenkins-module</packaging>
+
   <name>Instance Identity</name>
   <description>Maintains an RSA key pair that can serve as a foundation of authentication when communicating with Jenkins</description>
+  <licenses>
+    <license>
+      <name>MIT License</name>
+      <url>http://www.opensource.org/licenses/mit-license.php</url>
```
Comment (jglick) [inline pom.xml]: :ant: canonical URL is `https://opensource.org/licenses/MIT`

File: src/main/java/org/jenkinsci/main/modules/instance_identity/InstanceIdentity.java [hunk 3]
```diff
@@ -134,4 +136,27 @@ public static InstanceIdentity get() {
...
+                        .cn(Jenkins.getActiveInstance().getLegacyInstanceId())
+                        .o("instances")
+                        .ou("jenkins.io")
+                        .c("US")
+                        .validFrom(new Date(System.currentTimeMillis() - TimeUnit.DAYS.toMillis(1)))
+                        .validUntil(new Date(System.currentTimeMillis() + TimeUnit.DAYS.toMillis(366)))
+                        .sha256()
+                        .generate();
+            } catch (IOException e) {
+                return null;
```
Comment (jglick) [inline src/main/java/org/jenkinsci/main/modules/instance_identity/InstanceIdentity.java]: :ant: log it

**Comment by jglick** [comment]
:bee:

File: pom.xml (hunk 2 above)
Comment (stephenc) [inline pom.xml]: this was just a `mvn tidy:pom`

Comment (jglick) [inline pom.xml]: Sure, but so long as you are introducing diff hunks, might as well fix it.

**Pull request merged**